### PR TITLE
[9.18] Update repo URL and switch to GitHub Actions

### PR DESCRIPTION
**Pull request by GrahamCampbell**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,5 +1,4 @@
-/composer.lock
-/phpstan.neon
-/phpstan.phar
-/phpunit.xml
-/vendor
+composer.lock
+phpstan.neon
+phpunit.xml
+vendor
```
Comment (glensc) [inline .gitignore:4]: why going back to relative paths? if I have mistakenly created composer.lock in non-root dir, I would `git status` to show me that.

Comment (GrahamCampbell) [reply .gitignore:4]: Because these directories really do appear in places other than the root directory now, such as within the `vendor-bin` directory.

Comment (glensc) [reply .gitignore:4]: `/vendor-bin/*/vendor/` ... :)

altho, in my project I'll try just placing under vendor:
- https://github.com/bamarni/composer-bin-plugin#change-directory

```
"target-directory": "vendor/vendor-bin"
```

Comment (GrahamCampbell) [reply .gitignore:4]: But I need the composer.json files that are in the vendor-bin directory.

Comment (GrahamCampbell) [reply .gitignore:4]: Meh, I think what we have is OK. If there are any genuine issues caused, we can certainly revisit this.

**Pull request merged**